server/routes/user.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+from flask import Blueprint, current_app, session, abort, redirect, request
+from google.oauth2 import id_token
+from cachecontrol import CacheControl
+import google.auth.transport.requests
+
+bp = Blueprint('user', __name__)
```

Comment:
For these routing modules, I like having a url prefix that matches the file path so it's easy to know which module responds to a particular path.

Reply:
Done